CMIP6_source_id.json

```diff
@@ -7260,12 +7261,12 @@
         }
     },
     "version_metadata":{
-        "CV_collection_modified":"Fri Jan 15 17:45:43 2021 00000",
-        "CV_collection_version":"6.2.55.2",
+        "CV_collection_modified":"Tue Jan 19 16:16:44 2021 00000",
+        "CV_collection_version":"6.2.55.3",
         "author":"Matthew Mizielinski <matthew.mizielinski@metoffice.gov.uk>",
         "institution_id":"MOHC",
-        "previous_commit":"b1c65bc19f426e01c2a0596b2ffb811c503440e3",
-        "source_id_CV_modified":"Tue Dec 15 12:34:07 2020 -0800",
+        "previous_commit":"3ae4e3d4999af80c9bb90c23f31288498d65b162",
+        "source_id_CV_modified":"Fri Jan 15 17:45:43 2021 00000",
         "source_id_CV_note":"Revise source_id MPI-ESM1-2-HR",
```

Comment:
@matthew-mizielinski looks like `MPI-ESM1-2-HR` is noted here, whereas the edits were applied to `MPI-ESM1-2-LR`

Reply:
Apologies -- I'll check this more carefully next time.